src/CommonLib/hostfxr_utility.cpp

```diff
@@ -281,6 +281,7 @@ HOSTFXR_UTILITY::GetPortableHostfxrParameters(
     if (hFileHandle == INVALID_HANDLE_VALUE)
     {
         hr = ERROR_FILE_INVALID;
+        CloseHandle(hFileHandle);
```

Comment:
you cannot close a handle which has INVALID_HANDLE_VALUE. The handle was not created successfully yet.